v2/manifests/core-ons/zebedee.yml

```diff
@@ -1,45 +1,52 @@
-version: "3"
+version: "3.3"
 services:
   zebedee:
     build:
-      context: ../../../../zebedee
+      context: ${ROOT_ZEBEDEE:-../../../../zebedee}
       dockerfile: Dockerfile
+    expose:
+      - "8082"
     ports:
       - 8082:8082
     volumes:
-      - ../../../../zebedee-content:/zebedee_root
+      - ${zebedee_root}:/zebedee_root
     restart: unless-stopped
     environment:
-      DATASET_API_URL: $DATASET_API_URL
-      DATASET_API_AUTH_TOKEN: "FD0108EA-825D-411C-9B1D-41EF7727F465"
-      ENABLE_PERMISSIONS_AUTH: "true"
-      ENABLE_DATASET_IMPORT: "true"
-      FORMAT_LOGGING: "true"
-      JAVA_OPTS: " -Xmx1204m -Xdebug -Xrunjdwp:transport=dt_socket,address=8002,server=y,suspend=n"
-      PORT: "8082"
-      RESTOLINO_STATIC: "src/main/resources/files"
-      RESTOLINO_CLASSES: "zebedee-cms/target/classes"
-      PACKAGE_PREFIX: "com.github.onsdigital.zebedee"
-      audit_db_enabled: "false"
-      enable_splunk_reporting: "false"
-      zebedee_root: "/zebedee_root"
-      db_audit_url: "jdbc:postgresql://postgis:5432/audit"
-      db_audit_username: "postgres"
-      db_audit_password: "postgres"
-      publish_url: ""
-      ENABLE_CENTRALISED_KEYRING: "false"
-      KEYRING_SECRET_KEY: "38c03PzhNuSrYV8J0537XQ=="
-      KEYRING_INIT_VECTOR: "RkL9MmjfRcPB86alO82gHQ=="
-      SERVICE_AUTH_TOKEN: $SERVICE_AUTH_TOKEN
+      DATASET_API_URL:                ${DATASET_API_URL:-http://dp-dataset-api:22000}
+      DATASET_API_AUTH_TOKEN:         ${DATASET_API_AUTH_TOKEN:-FD0108EA-825D-411C-9B1D-41EF7727F465}
```

Comment:
As dp-compose is a public repo I would think that service auth token values should be held in secrets?

Reply:
I just made the value configurable via an env var, with a default value to the existing token.

I assumed this is the local value for `DATASET_API_AUTH_TOKEN` for local systems only (so it wouldn't be an issue exposing it), but if this value is used in any environment other than local, then it is already exposed by the existing code. I will leave a comment in slack to double check this.

This is also true for other vars like `AWS_COGNITO_...`

Reply:
Update: I decrypted secret values for `sandbox` and I validated that they are different 😌 
Maybe someone should also check prod values.

Reply:
I checked the staging and prod values. They look good.